ehrapy/tools/_sa.py

```diff
@@ -262,3 +262,15 @@ def anova_glm(result_1: GLMResultsWrapper, result_2: GLMResultsWrapper, formula_
     }
     dataframe = pd.DataFrame(data=table)
     return dataframe
+
+
+def cph(ad: AnnData, duration_col: str, event_col: str, entry_col: str = None) -> KaplanMeierFitter:
+    """Fit the Cox’s proportional hazard for the survival function.
+
+    See https://lifelines.readthedocs.io/en/latest/fitters/regression/CoxPHFitter.html
+    """
+    df = ad.to_df()
```

Comment:
Maybe you should use ehrapy's to_df function -> https://ehrapy.readthedocs.io/en/latest/usage/anndata/ehrapy.anndata.anndata_to_df.html#ehrapy.anndata.anndata_to_df